src-tauri/src/constants.rs

```diff
@@ -7,3 +7,21 @@ pub const MASTER_SERVER_URL: &str = "https://northstar.tf";
 
 // server list endpoint
 pub const SERVER_BROWSER_ENDPOINT: &str = "/client/servers";
+
+// List of core Northstar mods
+pub const CORE_MODS: [&str; 3] = [
```

Comment:
I don't remember the context for #120, but this PR is the occasion to solve it :)

Reply:
The context was that we declare core mods twice, once in frontend, once in backend which is just unnecessary duplication. As one cannot just do like 

```rust
#[ts(export)]
pub const BLACKLISTED_MODS: [&str; 3] = [
    "northstar-Northstar",
    "northstar-NorthstarReleaseCandidate",
    "ebkr-r2modman",
];
```

I'll have to add some function that frontend could call to get the list of core mods or other other constants. Will look into it but that shouldn't block this PR here, as this PR is basically a prerequisite for the other future one ^^

Reply:
Sorta addressed in #168